doc/api/fs.md

```diff
@@ -180,6 +180,10 @@ longer be used.
 <!-- YAML
 added: v10.0.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/41677
+    description: The `data` parameter no longer accepts objects with an
+                 own `toString` function property.
```

Comment:
In the end, did it ever accepted such objects? `node -e 'node -e 'fs.promises.open("/dev/null").then(fh => fh.write({ toString() { return "string" } }))'` results in an abort on all release lines (tested on v14.x, v16.x, and v17.x). If this was never supported, maybe we can simply fix the doc and remove the added entry (I realize you added this because I asked for it, sorry for the confusion, this `fs.md` file is too big, I got confused).